a_Calculate_Centers/src/calculate_AK_poi.R

```diff
@@ -0,0 +1,129 @@
+#' @title Calcuate POI center for NHD Best Resolution waterbodies from The 
+#' National Map
+#' 
+#' @description
+#' for the Best Resolution NHD file for the state of Alaska, download the 
+#' National Map file, subset to lakes/res/impoundments, subset to >= 1ha, and 
+#' calculate POI for each polygon
+#' 
+#' @returns silently saves .csv file in mid folder of the POI centers and 
+#' associated WBD metadata
+#' 
+#' 
+calculate_AK_poi <- function() {
+  # set timeout so that... this doesn't timeout
+  options(timeout = 60000)
+  # url for the NHD Best Resolutoin for 
+  url = "https://prd-tnm.s3.amazonaws.com/StagedProducts/Hydrography/NHD/State/GPKG/NHD_H_Alaska_State_GPKG.zip"
+  download.file(url, destfile = file.path("a_Calculate_Centers/nhd/", 
+                                          "Alaska_NHD.zip"))
+  unzip("a_Calculate_Centers/nhd/Alaska_NHD.zip", exdir = "a_Calculate_Centers/nhd/")
+  # remove zip
+  unlink("a_Calculate_Centers/nhd/Alaska_NHD.zip")
+  
+  # open the NHDWaterbody layer, coerce to a {sf} object
+  wbd <- st_read(file.path("a_Calculate_Centers/nhd/",
+                                 "NHD_H_Alaska_State_GPKG.gpkg"),
+                 layer = 'NHDWaterbody')
+  
+  # filter the waterbodies for ftypes of interest. 390 = lake/pond; 436 = res;
+  # 361 = playa
+  wbd <- wbd %>% 
+    filter(ftype %in% c(390, 436, 361)) 
+  
+  # filter for area > 1 hectare (0.01 km^2)
+  wbd <- wbd %>% 
+    filter(areasqkm >= 0.01) 
```

Comment:
suggestion: Same as in the other function

```suggestion
 wbd <- wbd %>% 
  filter(
    # filter the waterbodies for ftypes of interest. 390 = lake/pond; 436 = res;
    # 361 = playa
    ftype %in% c(390, 436, 361),
    # ...and for area > 1 hectare (0.01 km^2)
    areasqkm >= 0.01) 
```